src/views/PageSearchPortal.vue

```diff
@@ -90,31 +95,46 @@ export default {
         },
     },
     async created() {
-        const currentTab =
-            this.activeTab === 0 ? 'parking-request' : 'interested-request';
-        this.$router.push({
-            path: this.$route.fullPath,
-            query: { tab: currentTab },
-        });
-        this.getAgents();
-        const mobile = this.$route.query['mobile'];
-        if (mobile) {
-            this.updateMobileInput(mobile);
+        const tab = this.$route.query['tab'];
+        if (tab) {
+            if (tab === 'interested-request') {
```

Comment:
use tab names as string constants or may be as enum so that you can use that for setting active tab.